src/Application/Console/Formatters/CodeClimate.php

```diff
@@ -98,7 +101,9 @@ private function issues(InsightCollection $insightCollection, array $metrics): a
                                 'begin' => $detail->hasLine() ? $detail->getLine() : null,
                             ],
                         ],
-                        'category' => $climateCategories[$category],
+                        'categories' => [
```

Comment:
:sweat_smile: I think this change made a fail in phpstan inspection, as the return type is not the same